http-clients/src/test/java/org/triplea/http/client/SystemIdHeaderTest.java

```diff
@@ -14,18 +13,23 @@
 @ExtendWith(MockitoExtension.class)
 class SystemIdHeaderTest {
   private static final String SYSTEM_ID = "system-id";
-  private static final Supplier<SystemId> SYSTEM_ID_SUPPLIER = () -> SystemId.of(SYSTEM_ID);
+
+  private static SystemId systemIdSupplier() {
+    return SystemId.of(SYSTEM_ID);
```

Comment:
The method name seems out of date, perhaps we would just as well inline the supplier where it's needed?

Reply:
Resolved with: bbabe3f56fec05e1933454823389ba6e1dd99fde